src/particles/feedback_CIC_gpu.cu

```diff
@@ -21,7 +25,8 @@
 namespace supernova {
   curandStateMRG32k3a_t*  randStates;
   part_int_t n_states;
-  Real t_buff, dt_buff;
+  Real *d_snr, snr_dt, t_sn_start, t_sn_end;
```

Comment:
What does the `t_` prefix mean?

Reply:
time

Reply:
IMO it might be worth spelling out since `t` can be so many different things